### ISSUE TITLE
A11y_SQL database projects extension_Publish_New SQL Server_ScreenReader : NVDA is not announcing as Required (*) status, when focus is on "SQL server Port Number".

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/azuredatastudio#22134]
### GitHub Tags
#A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopApp; #ScreenReader; #WCAG1.3.1; #Win11; #SH-SQL Azure Data Studio-Win32-March23; #Win32; #SQL Azure Data Studio; #NVDA; #AILimited; #Info and Relationships;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

  ### Actual Result:​
NVDA is not announcing as Required (*) status, when focus is on "SQL server Port Number".

### Note issue: 
Similar issue observed in below scenarios.
1. Publish Project-> SQL Server admin Password
2. Publish Project->Confirm SQL Server admin Password
3. Punlish Project-> Base SQL Server Docker image 
4. Create new data base project ->Type
5. Create project from data base->Target project grouping->Location

### Expected Result:​
NVDA should announce the Required (*) status, when focus is on "SQL server Port Number".

### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.41.2 (user setup)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Screen reader: NVDA

### Repro Steps:

1. Launch Azure Data Studio
2. Tab till extension and search for “SQL Database Projects” and install it.
3. Tab till database project and press enter.
4. “Database Project” panel will open.
5. Tab till “View And More Action” button and press enter.
6. Tab till “Create Project From Database” and press enter.
7. Tab till server field and press enter.
8. “Connection” dialog will open. Fill the details for SQL server and connect.
9. Tab till Create button and press enter.
10. Database project will get create and will appear in Database project pane
11. Tab till created project name in Database Project panel.
12. Press context menu button(right click).
13. Tab till Publish and press enter.
14. Publish Project window will open.
15. Tab till “Publish to a new SQL server local development container” radio button under “Publish Target” and select it.
16. Prerequisite: have Docker installed on your machine.
17. Tab till port (select a port or keep the default ).
18. Enter a password that meets the password complexity guidelines. 
19. Tab till the docker image/ image tag to change it (if wanted). 
20. Tab till “I accept the Microsoft SQL Server License Agreement” checkbox and select it.
21. Tab till Publish button and press enter.
22. Verify whether the NVDA is announcing as Required (*) status, when focus is on "SQL server Port Number" or not.

### User Impact:
Screen reader users will not know which is the required field if screenreader is not announcing required status.

### Attachment:
![Presentation2](https://user-images.githubusercontent.com/124244478/222706908-79cb773f-1313-41a5-8cbf-3b04ed6dad47.gif)

![Req](https://user-images.githubusercontent.com/124244478/223643025-2049d586-637c-4efe-8049-3cd246837be6.gif)

**Comment by AnKushSingh05** [comment]
#Closed
Regressed the issue in the below environment
Version: 1.43.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/afafee844ce2b07eb55bdb6f40ee1ce88c0bbf4e
Date: 2023-03-29T05:28:54.263Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Issue is fixed. Hence closing the bug.
![Bug_22134_Fixed](https://user-images.githubusercontent.com/124244478/228767877-cb67d642-a6e0-4c8d-b7cc-b6f22a22c46a.gif)
